src/app/app.module.ts

```diff
@@ -50,20 +55,22 @@ import { BreadcrumbComponent } from "./breadcrumb/breadcrumb.component";
     SearchNavComponent,
     CscComponent,
     ProjectResourceComponent,
-    TitleLinkComponent,
     BreadcrumbComponent,
   ],
+  bootstrap: [AppComponent],
   imports: [
     LeafletModule,
     BrowserModule,
     FormsModule,
-    HttpClientModule,
     routing,
     NgSelectModule,
-    Ng2SmartTableModule,
     NgbDropdownModule,
     NgbModule,
     FontAwesomeModule,
+    MatTableModule,
+    MatSortModule,
+    MatCheckboxModule,
+    BrowserAnimationsModule,
   ],
   providers: [
     GoogleAnalyticsService,
```

Comment:
This cannot be in use!  Either delete now or create ticket.

Reply:
What do you mean they cannot be in use? They are all used to replace the ng2_smart_table and checkbox package that was deprecated.

Reply:
`GoogleAnalyticsService` can't be in use